src/duke/TaskList.java

```diff
@@ -35,6 +53,12 @@ public String markAsDone(int i) throws InvalidTaskNumberException{
                 "  " + t.toString();
     }
 
+    /**
+     * Removes a specified Task from the Task List.
+     * @param i Index of Task to be removed.
```

Comment:
There should be an empty line after the description of javadoc comments. Noticed a couple of this.